### ISSUE TITLE
VSI evaluation benchmark

### ISSUE THREAD

**Issue by SunghwanHong** [VITA-Group/VLM-3R#27]
Hi, 

Would it be possible to tell me where can I access/download  VSI benchmark for test? 
I can only find training set for VSI and VSTI, and evaluation set for VSTI.  

Thanks!

**Comment by Jian-Zhang-3DV** [comment]
Thanks for your question.

For a detailed introduction and access instructions for the VSI-Bench, please refer to the official project repository here:

[https://github.com/vision-x-nyu/thinking-in-space](https://github.com/vision-x-nyu/thinking-in-space)

The documentation in that repository should guide you on how to access the test set.

Hope this helps!
